pkg/collector/corechecks/servicediscovery/module/impl_linux_test.go

```diff
@@ -364,6 +365,78 @@ func buildFakeServer(t *testing.T) string {
 	return filepath.Dir(serverBin)
 }
 
+const (
+	pythonScript = `#! /usr/bin/python
```

Comment:
Should this be using python3 and/or with env? /usr/bin/python doesn't exist on my Ubuntu for example.

Reply:
changed to python3